.github/actions/bumpVersion/index.js

```diff
@@ -0,0 +1,9285 @@
+/**
```

Comment:
I wonder if `index.js` is the right name for this file? What about `dist.js`?

Reply:
As far as I can tell, there's no way to change the name of this file, only the directory it ends up in.

Running: `ncc build bumpVersion.js --out dist.js` results in this:

![image](https://user-images.githubusercontent.com/47436092/105220233-4a58c980-5b0c-11eb-9018-5b417864df82.png)